packages/bot-handler-avs/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "dependencies": {
+    "@types/long": "4.0.1",
+    "@types/moment-duration-format": "2.2.2",
```

Comment:
Are you sure you need those in the dependencies?

Reply:
this will be usually deployed in an AWS env and it has issues with dev deps, cause this project is going to be run directly with ts-node so it needs types...

Reply:
moment types are not needed, so I removed them.

Reply:
But the types for `moment-duration-format` are still there?